tests/test_fits.py

```diff
@@ -79,5 +80,31 @@ def test_fits_reading():
     im = imageio.imread(compressed)
     assert im.shape == (2042, 3054)
 
+@pytest.mark.skipif("astropy is None")
+def test_fits_get_reader():
+    """Test reading fits with get_reader method"""
+    
+    #Set precedence as normal
+    teardown_module()
+    
+    if IS_PYPY:
+        return  # no support for fits format :(
```

Comment:
I am aware that we do the same in other parts of the codebase, but it is actually an anti-pattern; likely from legacy code before pytest had a cleaner way to do things. We have to refactor this sooner or later ;)

Could you remove this and instead add the decorator `@pytest.mark.skipif("IS_PYPY", reason="pypy doesn't support astropy.fits.")`?

Reply:
Added this decorator as well.